single_call.yaml

```diff
@@ -7,10 +7,11 @@ config:
  
 scenarios:
   - name: "ETH Call"
+    beforeRequest: single_call
+    afterResponse: check_response
     flow:
-      - function: single_call
       - loop:
-        - post:
-            url: "/eth"
-            json: "{{ query }}"
+          - post:
```

Comment:
nanonit: indention seems off

Reply:
I purposefully indented here so that `count` and `loop` field keys appear to be "one level higher" than the list `[post]` and felt it was clearer to read this way. Basically, the JSON equivalent of this YAML is:
```json
{
  "loop": [
    {
      "post": {
         "url": "/eth",
         "json": "{{ query }}"
       },
    }
  ],
  "count": 50
}
```

Also, the extra indentation for list items is consistent with what we have in `scenario` and `flow` keys.

I can change it back if you prefer how it was before, no strong feelings.

Reply:
Let me know if you want me to make a follow up PR to address this, merging to avoid confusion WRT to running benchmarks on existing infra.